index.html

```diff
@@ -5,12 +5,34 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
+    <meta name="description" content="Portfolio website">
+    <meta name="keywords" content="HTML, CSS, Javascript">
+    <meta name="author" content="Maen Kh">
+    <link rel="stylesheet" href="/CSS/style.css">
     <title>Portfolio</title>
 </head>
 
 <body>
-
+    <!-- popup modal -->
+    <section class="modal-container">
+        <img class="modal-btn-close" src="/assets/images/modal-mobile-close.png" alt="">
+        <img class="modal-image" src="/assets/images/modal-mobile-img.png" alt="">
+        <div class="modal-title-btn-container">
+            <h1 class="modal-title">Keeping track of hundreds of components</h1>
+            <div class="modal-button">
+                <div class="btn btn-green"><a href="#">See Live <img class="icon-inside-button" src="/assets/images/modal-icon1.png" alt=""></a></div>
+                <div class="btn btn-green"><a href="#">See Source <img class="icon-inside-button" src="/assets/images/modal-icon2.png" alt=""></a></div>
+            </div>
```

Comment:
- [ ] Kindly in the mobile version the buttons are at the end of the modal, in your design they are at the top.

- [ ] Kindly make sure that the icons align the text, next to each other like in the Figma design.


| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/35535130/156625798-cab20ffc-2f71-4c50-a9de-5a5080e43fff.png) | ![image](https://user-images.githubusercontent.com/35535130/156625840-eec36baf-b371-4520-8ad2-2ce117a8ea12.png) |

Reply:
@keroloussamy Thank you for your helpful feedback.
I notice this difference but the only way to change the position of the button is javascript, because if use CSS absolute or relative the button will be cover the description text if this text becomes longer in some cases.
just if you can tell me what method should I use. Javascript or CSS.

thank you very much 😃

Reply:
Hello @maen1980, For this, you can give the section display: flex and use the [order](https://www.w3schools.com/cssref/tryit.asp?filename=trycss3_order) property to change the position for the buttons with the desktop version and the mobile version. And you can use gird also you can play with the column and rows to change the position between the mobile and desktop.